app/bundles/MessengerBundle/Config/services.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return function (ContainerConfigurator $configurator) {
+    $services = $configurator->services()
+        ->defaults()
+        ->autowire()
+        ->autoconfigure();
+
+    $services->load('Mautic\\MessengerBundle\\', '../')
+        ->exclude('../{'.implode(',', [
+                'Config',
+                'Tests',
+            ]).'}');
```

Comment:
```suggestion
        ->exclude('../{Config,Tests}');
```
might be more readable